pkg/chart/chart.go

```diff
@@ -18,10 +21,21 @@ type Metadata struct {
 }
 
 // Read reads a README YAML file and returns the Content to render.
-func ReadChartConfig(basePath string, chartName string) ([]byte, error) {
-	content, err := os.ReadFile(basePath + "/helm/" + chartName + "/README.md")
+func GenerateChartConfig(basePath string, chartName string) ([]byte, error) {
+	cmd := exec.Command("go", "install", "github.com/giantswarm/schemadocs@latest")
```

Comment:
i can't decide on installing this as part of the workflow - does it make sense to require it to be installed already? in case the user wants to use a different version or some such?

Reply:
yeah I included in the docskerfile